actions.go

```diff
@@ -103,6 +137,7 @@ type CustomActionsTriggerDefinitionUpdateInput struct {
 	AccessControl          CustomActionsTriggerDefinitionAccessControlEnum `json:"accessControl,omitempty"`
 	ResponseTemplate       *string                                         `json:"responseTemplate,omitempty"`
 	EntityType             CustomActionsEntityTypeEnum                     `json:"entityType,omitempty"`
+	ExtendedTeamAccess     *[]IdentifierInput                              `json:"extendedTeamAccess,omitempty"`
```

Comment:
This was the field I wanted extra tests on to ensure we can do the 3 states and that we generate the input variables correctly:
- not present doesn't show up in the variables
- empty array unsets any value already set on the trigger def
- array with value sets to just that value

We don't need to validate the response just that we can properly generate the necessary input for each of those states.